WalletWasabi/Models/EffectiveTransactionStatus.cs

```diff
@@ -0,0 +1,19 @@
+using System.Collections.Generic;
+using NBitcoin;
+
+namespace WalletWasabi.Models;
+
+public record Ancestor
+{
+	public uint256 txid { get; set; }
```

Comment:
Please use pascal. Also, use something like `AncestorCpfpTxInfo`  or similar. It would be good to use the compact notation `public record AncestorCpfpTxInfo(uint256 TxId, .....`